package/MDAnalysis/coordinates/TRZ.py

```diff
@@ -571,6 +571,8 @@ def _write_next_frame(self, obj):
         if ts.dimensions is not None:
             unitcell = triclinic_vectors(ts.dimensions).reshape(9)
         else:
+            warnings.warn("Timestep didn't have dimensions information, "
```

Comment:
We probably should have self-consistency here, if `None` == np.zeros(6), then `np.allclose(box, np.zeros(6))` should read in as `None` ? (my understanding is that currently it just sets a zero sized unitcell?)

Reply:
added a test for this, it's already implemented like this

Reply:
My bad I forgot about : https://github.com/MDAnalysis/mdanalysis/blob/4154e6304a38c076ef008907ed393d36ba5958a8/package/MDAnalysis/coordinates/base.py#L795

Side note - shouldn't that be an `allclose` [if we're comparing floats] ?